packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -152,7 +152,9 @@ export function qwikVite(qwikViteOpts: QwikVitePluginOptions = {}): any {
         resolveQwikBuild: true,
         transformedModuleOutput: qwikViteOpts.transformedModuleOutput,
         vendorRoots: [...(qwikViteOpts.vendorRoots ?? []), ...vendorRoots.map((v) => v.path)],
-        outDir: viteConfig.build?.outDir,
+        outDir: viteConfig.build?.assetsDir
+          ? (viteConfig.build?.outDir || CLIENT_OUT_DIR + '/') + viteConfig.build?.assetsDir
```

Comment:
shouldn't `CLIENT_OUT_DIR + '/'` be `path.join(CLIENT_OUT_DIR, '/')` instead, in order to support windows?

Reply:
Yep, you are right :man_facepalming:  Thanks for your feedback @IgorMinar appreciate